eventhubs/client/azure-eventhubs/src/test/java/com/azure/eventhubs/EventHubClientTest.java

```diff
@@ -48,12 +48,19 @@
     private final ClientLogger logger = new ClientLogger(EventHubClientTest.class);
 
     private EventHubClient client;
+    private EventHubProducer sender;
```

Comment:
We are abusing these member variables here. They can be set in multiple places from the same unit test. These should be local variables that take care of cleaning up themselves.